pkg/Microsoft.Windows.Compatibility/test/Microsoft.Windows.Compatibility.Validation.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+  	<PreserveCompilationContext>true</PreserveCompilationContext>
+  </PropertyGroup>
```

Comment:
You probably want to hard-code the 2.0.0 version of Microsoft.NETCore.App in the project. By the way just passing TargetFramework as netcoreapp2.0 doesn't always guarantee the 2.0.0 of Microsoft.NETCore.App so it might be worth explicitly setting the package version as well.

Reply:
I will do that then. Thanks for the heads up.

Reply:
Adding the package 2.0.0 causes a warning when publishing for netcoreapp2.0:

```
E:\repos\corefx\Tools\dotnetcli\sdk\2.0.0\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.DefaultItems.targets(199,5): warning : A PackageReference for 'Microsoft.NETCore.App' was included in your project. This package is implicitly referenced by the .NET SDK and you do not typically need to reference it from your project. For more information, see https://aka.ms/sdkimplicitrefs [E:\repos\corefx\pkg\Microsoft.Windows.Compatibility\tests\Microsoft.Windows.Compatibility.Validation.csproj]
```